arches/app/models/models.py

```diff
@@ -1702,7 +1702,7 @@ class TileModel(SaveSupportsBlindOverwriteMixin, models.Model):  # Tile
         related_query_name="child",
     )
     data = JSONField(blank=True, default=dict, db_column="tiledata")
-    nodegroup_id = models.UUIDField(db_column="nodegroupid", null=True)
+    nodegroup_id = models.UUIDField(db_column="nodegroupid", null=True, db_index=False)
```

Comment:
No change here (db_index is False by default--notice no migration was generated), but I left it in to reinforce the comment below.